guidelines/MIGRATING.md

```diff
@@ -0,0 +1,24 @@
+# Migrating ACPaaS UI
+
+## v3 to v4
+The v4 of ACPaaS UI introduces a number of breaking changes, the most important being:
+- The minimum Angular version was changed to 8
+- The packages are no longer scoped under @acpaas-ui/ngx-packages/package, but @acpaas-ui/ngx-package.
+
+To succesfully migrate a project:
+- Update your Angular version following the Angular [upgrade documentation](https://update.angular.io/).
+- Remove the `@acpaas-ui/ngx-components' dependency from your package.json, as it has been replaced.
+- Install all ACPaaS UI components as separate packages.
```

Comment:
Lijkt me op zich zeker geen slecht uitgangspunt om de packages individueel te publiceren. 👍 Dat geeft wel wat meer complexiteit omtrent release cycles en version management. Hebben jullie daar al over nagedacht? Gaan jullie de versies bijvoorbeeld hetzelfde houden tussen de packages zoals Angular het zelf doet of gaan jullie ze apart laten lopen zoals bij de oude acpaas-ui?